Makefile

```diff
@@ -0,0 +1,185 @@
+# Environment checkup
+ifndef IMAGE_DIR
+$(error "No image directory specified (IMAGE_DIR)")
+endif
+include $(IMAGE_DIR)/env.mk
+ifndef IMAGE_NAME
+$(error "No image base name found (IMAGE_NAME), e.g. 'ubuntu'")
+endif
+ifndef IMAGE_VERSION
+$(error "No image version found (IMAGE_VERSION), e.g. 'xenial'")
+endif
+ifndef IMAGE_TITLE
+$(error "No image title found (IMAGE_TITLE), e.g. 'Ubuntu Xenial (16.04)'")
+endif
+
+DOCKER_NAMESPACE ?= scaleway
+BUILD_OPTS ?=
+SERVE_ROOTFS ?= y
+REGION ?= par1
+
+# Architecture variables setup
+HOST_ARCH := $(shell uname -m)
+ARCH ?=	$(HOST_ARCH)
+ifneq ($(ARCH), $(HOST_ARCH))
+$(shell docker run --rm --privileged multiarch/qemu-user-static:register --reset)
```

Comment:
That doesn't respect `ARCH=amd64`  yet.
The normalization of the `TARGET_QEMU_ARCH` on https://github.com/scaleway/image-tools/pull/196/files#diff-b67911656ef5d18c4ae36cb6741b7965R62 should happen earlier

Reply:
Looks like bash doesn't allow output in here.
You should use `$(shell docker run --rm --privileged multiarch/qemu-user-static:register --reset > /dev/null)`

Reply:
I've tried fixing all that and making sure everything is normalized before anything is built, is that better ?